versions.tf

```
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = "= 1.14.1"
+    }
+  }
+}
```

Notice the version constraint "~>" to allow new patch releases within a specific minor release. 
This is not mandatory but, we are including `provider_meta` in all of our modules so that we can update the user agent accordingly, it allows us to keep track of module usage.

More info https://github.com/equinix-labs/equinix-labs/blob/main/terraform-module-standards.md#equinix-provider

```suggestion
terraform {
  required_version = ">= 0.13"
  required_providers {
    equinix = {
      source  = "equinix/equinix"
      version = "~> 1.14"
    }
  }
  provider_meta "equinix" {
    module_name = "terraform-equinix-network-edge-nginx"
  }
}
```